src/tips/models.py

```diff
@@ -24,18 +84,24 @@ class TipType(models.Model):
     def can_show(self, page: str, version_count: int, user: ClusiveUser):
         """Test whether this tip can be shown on a particular page"""
         # Teacher/parent-only tips
-        if user.role == Roles.STUDENT and self.name in ['book_actions', 'activity']:
+        if user.role == Roles.STUDENT and self.name in TEACHER_ONLY_TIPS:
```

Comment:
I should have included a GUEST user in the tests.  When GUEST is tested, the `TipType.can_show()` fails for the `activity` tip on the Dashboard.

Interestingly, it should also fail with the current version  in production where only `book_actions` and `activity` tips are checked.  Do `book_actions` or `activity` tooltip pop-up for guests?  I suspect not because the Dashboard for guests does no include the relevant activity panel nor the book_actions menu.  As such, there is nothing for the tooltip to point to.

But, in the current situation, there are more `TEACHER_ONLY_TIPS` some of whose targets appear for guest users.  So, I'll check for guest users.